antarest/main.py

```diff
@@ -75,10 +76,28 @@ def get_local_path() -> Path:
         return Path(os.path.abspath(""))
 
 
+def configure_logger(config: ConfigYaml) -> None:
```

Comment:
Use `ConfigYaml` is an implementation of `Config` class. Therefore we should use `Config` as type.